Composer/packages/tools/language-servers/language-generation/src/LGServer.ts

```diff
@@ -43,6 +45,7 @@ export class LGServer {
   protected readonly documents = new TextDocuments();
   protected readonly pendingValidationRequests = new Map<string, number>();
   protected LGDocuments: LGDocument[] = []; // LG Documents Store
+  private readonly memoryVariables: object;
```

Comment:
this should be able to be updated. 

this PR's goal is figure out a way to dynamic set and update this stuff and auto-suggest based on this data source.